app/views/tattoos/show.html.erb

```diff
@@ -45,8 +50,9 @@
   <% end %>
 </div>
 
+<div class="section-box panel large-6 medium-8 small-12 large-centered small-centered columns">
 <h4>Rate this Ink</h4>
-  <div class="review_box small-12 medium-6 large-4">
+  <div class="review_box small-12 medium-12 large-12">
```

Comment:
If you're using the same column width for all screen sizes, just do `small-12` - medium and large will by default just be whatever `small` is set to!